R-package/tests/testthat/test_feature_weights.R

```diff
@@ -0,0 +1,33 @@
+library(xgboost)
+
+context("feature weights")
+
+n_threads <- 2
+
+test_that("training with feature weights works", {
```

Comment:
I'd say better not to add extra tests for this short-lived release, since they increase the chances of rejections and extra fixes being needed.

Reply:
This is merely restoring the tests removed in the previous commit.